test/OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests/IntegrationTests.cs

```diff
@@ -149,7 +149,7 @@ public void TraceExportResultIsSuccess(OtlpExportProtocol protocol, string endpo
         [SkipUnlessEnvVarFoundTheory(CollectorHostnameEnvVarName)]
         public void MetricExportResultIsSuccess(OtlpExportProtocol protocol, string endpoint, bool useManualExport, bool forceFlush)
         {
-#if NETCOREAPP3_1
+#if NET6_0_OR_GREATER
```

Comment:
Same as above.

Reply:
Good catch. I think the right thing to do here is just delete these whole blocks. There is no `netcoreapp3.1` target for the `OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests` and `OpenTelemetry.Instrumentation.Grpc.Tests` projects anymore. We no longer need this code.

Reply:
I will do that as follow up. trying to keep code changes separate :). For now, I have just added it back